charts/rancher-vsphere-csi/values.yaml

```diff
@@ -25,22 +25,22 @@ csiController:
     enabled: false
   image:
     repository: rancher/mirrored-cloud-provider-vsphere-csi-release-driver
-    tag: v2.6.2
+    tag: latest
```

Comment:
Set the default tags to `latest` to make it obvious when the version overrides are not working properly